doc/content/gateways/lora-basics-station/cups.md

```diff
@@ -19,17 +19,58 @@ weight: -1
 
 To connect a gateway using the CUPS protocol, you must first add the gateway in {{% tts %}}. This can be done either in the console, or via the command line. See instructions for [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}). 
 
-## Create an API Key
+## Create Separate CUPS and LNS API Keys
+
+Since CUPS automatically configures LNS, you will need two API keys.
 
 CUPS requires an API key for your gateway with the following rights:
 - View gateway information
 - Edit basic gateway settings
 
+LNS requires an API Key with the following rights:
+- Link as Gateway to a Gateway Server for traffic exchange, i.e. write uplink and read downlink
+
 To create an API key for your gateway, follow instructions for Creating a Gateway API key in [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}).
 
+## Configure CUPS to Send the LNS API Key
+
+{{% cli-only %}}
+
+We need to configure CUPS in {{% tts %}} to transmit the LNS API key when a gateway connects. Use the following command to do so, replacing `"your-gateway-id"` with your gateway ID in {{% tts %}} and  `"your-lns-api-key"` with the LNS API key you created in the last step:
+
+```bash
+$ export GTW_ID="your-gateway-id"
+$ export LNS_KEY="your-lns-api-key"
+$ export SECRET=$(echo $LNS_KEY | xxd -p -u | perl -pe 's/\n//')
+$ ttn-lw-cli gateways update $GTW_ID --lbs-lns-secret.value $SECRET
+```
+
+If successful, you should receive a response as follows:
+
+<details>
+<summary>Show CLI output</summary>
+
+```json
+{
+  "ids": {
+    "gateway_id": "<gateway-id>"
+  },
+  "created_at": "2020-10-13T10:49:02.730Z",
+  "updated_at": "2020-11-17T14:52:06.440Z",
+  "version_ids": {
+
+  },
+  "lbs_lns_secret": {
+    "key_id": "is/gateway-secrets-encryption-key",
+    "value": "<base64-lns-api-key>"
```

Comment:
Also this is the encrypted value that's returned. 
If you want the actual secret, then 
```
ttn-lw-cli gateways get $GTW_ID --lbs-lns-secret.value
```